src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-weight: 300;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.top__bar__search {
+  box-sizing: border-box;
+  display: flex;
+  margin-top: 20px;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  width: 100%;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  align-items: center;
+
+}
+
+.top__bar__search--small {
+  height: 42px;
+}
+
+.top__bar__search:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+  border-radius: 4px;
+  background: #fff;
+
+}
+
+.top__bar__search:focus {
```

Comment:
Consider applying this styling for the input element instead of the form. So some styles should be moved to the input field. Also, you can add the position relative to the form to place an icon properly